tests/e2e/specs/Multiselect.spec.js

```diff
@@ -122,4 +122,45 @@ describe('Multiselect', () => {
       });
     });
   });
+
+  it('should not move multiple shapes when shift + combo was held down', () => {
+    [
+      {combo: 'meta', task1Position: { x: 100, y: 100 }},
+      {combo: 'ctrl', task1Position: { x: 400, y: 100 }},
+      {combo: 'alt', task1Position: { x: 700, y: 100 }},
+    ].forEach(({combo, task1Position}) => {
+      const task2Position = { x: task1Position.x + 100, y: task1Position.y + 100 };
+      const translateAmount = 200;
+      const newTask1Position = { x: task1Position.x + translateAmount, y: task1Position.y + translateAmount };
+
+      dragFromSourceToDest(nodeTypes.task, task1Position);
+      dragFromSourceToDest(nodeTypes.task, task2Position);
+
+      cy.get('.paper-container').click();
+
+      cy.get('body').type(`{shift}{${ combo }}`, { release: false });
+      cy.get('.paper-container').as('paperContainer').trigger('mousedown', 'topLeft');
+      cy.get('@paperContainer').trigger('mousemove', 'bottomRight');
+      waitToRenderAllShapes();
+      cy.get('@paperContainer').trigger('mouseup');
+      cy.get('body').type(`{shift}{${ combo }}`, { release: true });
+
+      moveElementRelativeTo(task1Position, translateAmount, translateAmount, nodeTypes.task);
+
+      waitToRenderAllShapes();
+
+      getElementAtPosition(newTask1Position, nodeTypes.task).then($task => {
+        getGraphElements().then(elements => {
+          const { x, y } = elements.find(el => el.get('id') === $task.attr('model-id')).position();
+          expect({ x, y }).to.eql(newTask1Position);
+        });
+      });
+      getElementAtPosition(task2Position, nodeTypes.task).then($task => {
+        getGraphElements().then(elements => {
+          const { x, y } = elements.find(el => el.get('id') === $task.attr('model-id')).position();
+          expect({ x, y }).to.eql(task2Position);
+        });
+      });
```

Comment:
When I comment out your fix (lines 46-49 in `setUpSelectionBox.js`), this test fails, however, it doesn't fail due to the assertion, but because of a type error:
```
TypeError: Cannot read property 'attr' of null
```

So if this behaviour is broken in the future, and this test fails the same way, it may not be obvious why exactly its broken or how to fix it. I think if the fix is commented out, the test should fail with an assertion.

To keep this test small and focused, maybe you don't have to add elements to the page, maybe trigger a <kbd>shift</kbd> key press with the combo buttons, a mouse down + move (but no mouse up), and then assert the cursor is the default one, and that no selection box is visible.